documentation/library_guide/kernel_templates/esimd_main.rst

```diff
@@ -15,3 +17,16 @@ The following algorithms are implemented:
    :hidden:
 
    esimd/radix_sort
+
+-------------------
+System Requirements
+-------------------
+
+- Hardware: Intel® Data Center GPU Max Series.
+- Compiler: Intel® oneAPI DPC++/C++ Compiler 2023.2 and newer.
+- Operating Systems:
+
+  - Red Hat Enterprise Linux* 9.2,
+  - SUSE Linux Enterprise Server* 15 SP5,
+  - Ubuntu* 22.04.
+  - Other distributions and their versions listed in `<https://dgpu-docs.intel.com/driver/installation.html>`_ should be supported accordingly, however they have not been tested.
```

Comment:
```suggestion
  - Other distributions and their versions listed in `<https://dgpu-docs.intel.com/driver/installation.html>`_ (testing may be limited or not performed).
```
I am not sure though if something like this is appropriate at all in our documentatiion.

Reply:
Let's remove that line entirely. It will have the same what oneAPI system requirements have: https://www.intel.com/content/www/us/en/developer/articles/system-requirements/intel-oneapi-base-toolkit-system-requirements.html#:~:text=Intel%C2%AE%20Data%20Center%20GPU%20Max%20Series%20(PVC,Ubuntu%2022.04